source/_components/ffmpeg.markdown

```diff
@@ -28,6 +28,18 @@ Configuration variables:
 - **ffmpeg_bin** (*Optional*): Set the ffmpeg binary (eg. `/usr/bin/ffmpeg`). Default 'ffmpeg'. 
 - **run_test** (*Optional*):  Check if `input` is usable by ffmpeg. Default True.
 
+### {% linkable_title Debian Jessie Installations %}
+To get the binary on Debian Jessie on a RPi you need to perform the following:
```

Comment:
Are you referring to Debain Jessie or Raspbian? Raspbian does some changes to Debian and depending on the base in question your instructions may fail.